tests/test_graphical_units/img_svg_resources/multiple_transforms.svg

```diff
@@ -0,0 +1,636 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
I presume this test's result was visually inspected for correctness?

Reply:
correct